notebooks/a01_manifold_class.ipynb

```diff
@@ -0,0 +1,2260 @@
+{
```

Comment:
<p>one-dimensional circle and two-dimensional sphere: the dimension of the manifold is equal to the number of degrees of freedom of a point moving on that manifold.</p><p> </p><p>NIT: that are at a given distance</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1340/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>done</p>